{{cookiecutter.project_slug}}/requirements/environment.yml

```diff
@@ -0,0 +1,42 @@
+# This file may be used to create an environment using:
```

Comment:
Does it make sense to provide a non-empty `environment.yml` in the blueprint, which will necessarily be out-of-date when a new project is created? I'd suggest to provide empty `*environment.yml` files and leave it to the user to pin their environment (they'll have to do so anyway once they add runtime dependencies to the project). (But I don't have particularly strong feelings about this.)